draft-ietf-quic-transport.md

```diff
@@ -1336,11 +1335,7 @@ and sets the SCIL field to indicate the length.
 The first flight of 0-RTT packets use the same Destination and Source Connection
 ID values as the client's first Initial.
 
-The Destination Connection ID field in the server's Initial packet contains a
-connection ID that is chosen by the recipient of the packet (i.e., the client);
-the Source Connection ID includes the connection ID that the sender of the
-packet wishes to use (see {{connection-id}}). The server MUST use consistent
-Source Connection IDs during the handshake.
+The server MUST use consistent Source Connection IDs during the handshake.
```

Comment:
This implies that the server cannot change its Source Connection ID between sending a Retry and sending an Initial.  I hope that we agree that this is an implausible requirement.  The consistency applies only after sending an Initial.

Reply:
Weirdly, this text was already there, see lines 1342/1343.

But I agree, this is clearly incorrect or depends upon the definition of consistent, which I don't believe is worth defining.  I'm going to remove this paragraph.